examples/flutter_gallery/lib/demo/cupertino/cupertino_segmented_control_demo.dart

```diff
@@ -0,0 +1,117 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+const Color _kKeyUmbraOpacity = const Color(0x33000000); // alpha = 0.2
+const Color _kKeyPenumbraOpacity = const Color(0x24000000); // alpha = 0.14
+const Color _kAmbientShadowOpacity = const Color(0x1F000000); // alpha = 0.12
+
+class CupertinoSegmentedControlDemo extends StatefulWidget {
+  static const String routeName = 'cupertino/segmented_control';
+
+  @override
+  _CupertinoSegmentedControlDemoState createState() => new _CupertinoSegmentedControlDemoState();
+}
+
+class _CupertinoSegmentedControlDemoState extends State<CupertinoSegmentedControlDemo> {
+  final Map<int, Widget> children = const <int, Widget>{
+    0: Center(
+      child: const Text('Red'),
```

Comment:
Feel free to have some fun with the names.

For example: "Glowing" (for Red) "Verdant" (for Green) "Cerulean" (for Blue).

Or whatever you think would amuse people.